vaadin-dir-helper.html

```diff
@@ -0,0 +1,97 @@
+<!--
+@license
+Copyright (c) 2020 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<script>
+  window.Vaadin = window.Vaadin || {};
+
+  Vaadin.DirHelper = class VaadinDirHelper {
```

Comment:
I'm not sure but this potentially could leak reference to `VaadinDirHelper` class globally.

Please consider wrapping into IIFE same as we do here in [Vaadin.FocusablesHelper](https://github.com/vaadin/vaadin-overlay/blob/master/src/vaadin-focusables-helper.html)

Reply:
Updated.